demos/browser/app/meeting/CloudWatchLogger.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import LogLevel from '../../../../src/logger/LogLevel';
+import {ConsoleLogger} from '../../../../src/index';
+
+export default class CloudWatchLogger extends ConsoleLogger {
```

Comment:
This implementation should go in the SDK (logger/CloudWatchLogger) so it can be reused